### ISSUE TITLE
Homepage haiku isn't 5-7-5

### ISSUE THREAD

**Issue by nbibler** [haml/haml-site#8]
``` haml
%h1
  Beautiful, DRY,<br />
  well-indented, clear markup:<br />
  %strong templating haiku.
```

From Wikipedia:

> Traditional haiku consist of 17 on, in three phrases of five, seven and five on respectively. Among contemporary poems teikei (定型 fixed form) haiku continue to use the 5-7-5 pattern while jiyuritsu (自由律 free form) haiku do not.[citation needed] One of the examples below illustrates that traditional haiku masters were not always constrained by the 5-7-5 pattern.

Presuming the intent was to follow a common haiku format, the current haiku only represents a 4-7-5 meter. Might I suggest a couple of possible corrections to return to the traditional 5-7-5:

```
Beautiful and DRY,
well-indented, clear markup:
templating haiku.
```

```
Beautifully DRY,
well-indented, clear markup:
templating haiku.
```

**Comment by KennethWilke** [comment]
I always get stuck on this part of the website, then I can't read any more and close it

**Comment by nbibler** [comment]
Exactly. :grinning:

**Comment by PooSham** [comment]
Wow, this issue is over 3 years old and still not fixed. This really irritated me when I saw it.

**Comment by test-as-other-user** [comment]
I noticed this too, and immediately Google searched "haml haiku isn't a haiku" and found this issue.

I can't believe they actually think this doesn't matter. It's sloppy aesthetics! 🤦

**Comment by KennethWilke** [comment]
They fixed it! I was talking about this today and checked the site and it is now a legit haiku

**Comment by nbibler** [comment]
This appears to have been addressed in e45297a via #10.
